task/thermal/src/bsp/gimlet_bcd.rs

```diff
@@ -24,12 +24,20 @@ task_slot!(SEQ, gimlet_seq);
 // the control loop.
 const NUM_TEMPERATURE_SENSORS: usize = sensors::NUM_TMP117_TEMPERATURE_SENSORS;
 
+#[cfg(any(target_board = "gimlet-b", target_board = "gimlet-c"))]
+const NUM_NVME_BMC_TEMPERATURE_SENSORS: usize =
+    sensors::NUM_NVME_BMC_TEMPERATURE_SENSORS
+        + sensors::NUM_M2_HP_ONLY_TEMPERATURE_SENSORS;
+
+#[cfg(not(any(target_board = "gimlet-b", target_board = "gimlet-c")))]
```

Comment:
Nit: maybe specify as `target_board = "gimlet-d"` here and below?  It's shorter _and_ and has the advantage of complaining in the future when we add new targets.

Reply:
So, the rationale there was that future Gimlet boards really shouldn't have this issue, and I didn't want to necessitate changing this if/when we have another Gimlet rev -- and then didn't want to run the risk that anything would silently compile. But perhaps that's a misplaced concern?

Reply:
Eh, I think it's better to call out `gimlet-d` explicitly.

Reply:
Fixed.